docs/content/connectors/oceanbase-cdc.md

```diff
@@ -0,0 +1,506 @@
+
+   ```shell
+   mysql -h${host} -P${port} -uroot
+   
+   mysql> SHOW TENANT;
+   mysql> CREATE USER ${sys_username} IDENTIFIED BY '${sys_password}';
+   mysql> GRANT ALL PRIVILEGES ON *.* TO ${sys_username} WITH GRANT OPTION;
+   ```
+
+3. Create a user in the tenant you want to monitor, this is used to read data for snapshot and change event.
+
+4. Get the `rootservice_list`. You can use the following command to get the value:
+
+    ```shell
+    mysql> show parameters like 'rootservice_list';
+    ```
+
+5. Setup OceanBase LogProxy following the [quick start](https://github.com/oceanbase/oblogproxy#quick-start).
+
+How to create a OceanBase CDC table
+----------------
+
+The OceanBase CDC table can be defined as following:
+
+```sql
+-- checkpoint every 3000 milliseconds                       
+Flink SQL> SET 'execution.checkpointing.interval' = '3s';
+
+-- register a OceanBase table 'orders' in Flink SQL
+Flink SQL> CREATE TABLE orders (
+    order_id     INT,
+    order_date   TIMESTAMP(0),
+    customer_name STRING,
+    price        DECIMAL(10, 5),
+    product_id   INT,
+    order_status BOOLEAN,
+    PRIMARY KEY (order_id) NOT ENFORCED
+) WITH (
+    'connector' = 'oceanbase-cdc',
+    'scan.startup.mode' = 'initial',
+    'username' = 'user@test_tenant',
+    'password' = 'pswd',
+    'tenant-name' = 'test_tenant',
+    'database-name' = 'test_db',
+    'table-name' = 'orders',
+    'hostname' = '127.0.0.1',
+    'port' = '2881',
+    'rootserver-list' = '127.0.0.1:2882:2881',
+    'logproxy.host' = '127.0.0.1',
+    'logproxy.port' = '2983');
+
+-- read snapshot and binlogs from orders table
+Flink SQL> SELECT * FROM orders;
+```
+
+Connector Options
+----------------
+
+<div class="highlight">
+    <table class="colwidths-auto docutils">
+        <thead>
+            <tr>
+                <th class="text-left" style="width: 10%">Option</th>
+                <th class="text-left" style="width: 8%">Required</th>
+                <th class="text-left" style="width: 7%">Default</th>
+                <th class="text-left" style="width: 10%">Type</th>
+                <th class="text-left" style="width: 65%">Description</th>
+            </tr>
+        </thead>
+        <tbody>
+            <tr>
+                <td>connector</td>
+                <td>required</td>
+                <td style="word-wrap: break-word;">(none)</td>
+                <td>String</td>
+                <td>Specify what connector to use, here should be <code>'oceanbase-cdc'</code>.</td>
+            </tr>
+            <tr>
+                <td>scan.startup.mode</td>
+                <td>required</td>
+                <td style="word-wrap: break-word;">(none)</td>
+                <td>String</td>
+                <td>Specify the startup mode for OceanBase CDC consumer, valid enumerations are
+                    <code>'initial'</code>,<code>'latest'</code> or <code>'timestamp'</code>.
```

Comment:
This is related to the next option `scan.startup.timestamp`. Do you think we should them together? 
Since it's actually a number of timestamp here, using `scan.startup.latest-offset` seems a little weird.